spec/unit/repository/repository-fetch.spec.ts

```diff
@@ -99,5 +99,70 @@ describe("Repository", () => {
       it("returns the expected entity", () =>
         expect(entity).toEqual(expect.objectContaining({ entityId, ...data.expectedData })));
     });
+
+    //#region fetchMany
```

Comment:
Regions might be a good idea, but let's do it holistically as a single commit, not as part of this. Also, please remove the comment on 104.  Thanks!

Reply:
i will, it was just a guide for me that i forgot to delete my bad